### PR TITLE
1.2 only: add a note to manually move pods if necessary

### PR DESCRIPTION
**Pull request by dzou-hpe**
## Summary and Scope

Add manual check before we rebuild/upgrade a worker node.  Note that this is only for 1.2 and in future releases, this issue would be addressed by code.

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves CASMINST-4748
* Change will also be needed in `<insert branch name here>`
* Future work required by [issue id](issue link)
* Documentation changes required in [issue id](issue link)
* Merge with/before/after `<insert PR URL here>`

## Testing

_List the environments in which these changes were tested._

### Tested on:

  * `<development system>`
  * Local development environment
  * Virtual Shasta

### Test description:

_How were the changes tested and success verified? If schema changes were part of this change, how were those handled in your upgrade/downgrade testing?_

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)?
- Were continuous integration tests run? If not, why?
- Was upgrade tested? If not, why?
- Was downgrade tested? If not, why?
- Were new tests (or test issues/Jiras) created for this change?

## Risks and Mitigations

_Are there known issues with these changes? Any other special considerations?_

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ ncn# echo $XNAME
 Only follow the steps in the section for the node type that is being rebuilt:
 
 * Worker node
+  
+  1. Make sure not all pods of `ingressgateway-hmn, spire-server` are running on the same worker nodes. If all pods of above deployments are placed onto one worker node, then use `/opt/cray/platform-utils/move_pod.sh` to move at least one pod to a different worker noede.
```
Comment (mharding-hpe) [inline operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md:43]:
```suggestion
    1. Make sure that not all pods of `ingressgateway-hmn` or `spire-server` are running on the same worker node.
  
          For either of those two deployments, if all pods are running on a single worker node, then use the
          `/opt/cray/platform-utils/move_pod.sh` script to move at least one pod to a different worker node.
```

File: operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md [hunk 2]
```diff
@@ -39,6 +39,8 @@ ncn# echo $XNAME
...
```
Comment (mharding-hpe) [inline operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md:44]:
```suggestion

    1. Rebuild the worker node.

```

File: operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md [hunk 3]
```diff
@@ -39,6 +39,8 @@ ncn# echo $XNAME
...
 
   ```bash
```
Comment (mharding-hpe) [inline operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md:45]:
```suggestion
        ```bash
```

File: operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md [hunk 4]
```diff
@@ -39,6 +39,8 @@ ncn# echo $XNAME
...
   ncn-m001# /usr/share/doc/csm/upgrade/1.2/scripts/rebuild/ncn-rebuild-worker-nodes.sh ncn-w001
```
Comment (mharding-hpe) [inline operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md:46]: The line after this one also needs to be indenter further
```suggestion
        ncn-m001# /usr/share/doc/csm/upgrade/1.2/scripts/rebuild/ncn-rebuild-worker-nodes.sh ncn-w001
```

File: upgrade/1.2/Stage_2.md [hunk 5]
```diff
@@ -13,6 +13,7 @@
 1. Repeat the previous step for each other master node **excluding `ncn-m001`**, one at a time.
 
 ## Stage 2.2
```
Comment (mharding-hpe) [inline upgrade/1.2/Stage_2.md:15]:
```suggestion
## Stage 2.2

```

File: upgrade/1.2/Stage_2.md [hunk 6]
```diff
@@ -13,6 +13,7 @@
...
+1. Make sure not all pods of `ingressgateway-hmn, spire-server` are running on the same worker nodes. If all pods of above deployments are placed onto one worker node, then use `/opt/cray/platform-utils/move_pod.sh` to move at least one pod to a different worker noede.
```
Comment (mharding-hpe) [inline upgrade/1.2/Stage_2.md:16]:
```suggestion
1. Make sure that not all pods of `ingressgateway-hmn` or `spire-server` are running on the same worker node.
  
      For either of those two deployments, if all pods are running on a single worker node, then use the
      `/opt/cray/platform-utils/move_pod.sh` script to move at least one pod to a different worker node.
```

**Review by mharding-hpe — CHANGES_REQUESTED** [review]

**Review by mharding-hpe — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Pull request merged**